mobile-widgets/qml/CloudCredentials.qml

```diff
@@ -144,5 +144,12 @@ Item {
 				}
 			}
 		}
+		SsrfButton {
+			id: signin_forgot_password
+			text: qsTr("Forgot password?")
```

Comment:
that's an appropriate text for a button
We could have longer text above or below it.
I fear that the text that I have in the menu now is too long for narrow screens - yet it's much more precise on what this link does.